tests/integration/test_simplified_workflows.py

```
@@ -0,0 +1,556 @@
+        """Verify required workflow keys are present."""
+        assert 'name' in pr_agent_workflow
+        assert 'on' in pr_agent_workflow
+        assert 'jobs' in pr_agent_workflow
+    
+    def test_pr_agent_workflow_name(self, pr_agent_workflow):
+        """Verify workflow has a descriptive name."""
+        assert pr_agent_workflow['name']
+        assert len(pr_agent_workflow['name']) > 0
+        assert isinstance(pr_agent_workflow['name'], str)
+    
+    def test_pr_agent_no_context_chunking_references(self, pr_agent_workflow):
```

Ruff (0.14.6)

Unused method argument: `pr_agent_workflow`

(ARG002)